CHANGELOG.md

```diff
@@ -1,12 +1,10 @@
-## 2.3.0-beta.2
-- fix: Adds additional diagnostic information to database query exceptions.
-- fix: Resolved an issue that caused premature closure of method stream websocket connections.
-
-## 2.3.0-beta.1
+## 2.3.0
```

Comment:
Comment: Let me bump latest version of dart major in main before merging this PR! :)